prompting/datasets/wiki.py

```diff
@@ -90,8 +75,8 @@ def get_article_sections(title: str) -> Dict:
 
 
 def process_page(
-    page, exclude_sections: Optional[list] = None, valid_section: callable = None, selector: Selector = None
-) -> Dict:
+    page: wikipedia.WikipediaPage, exclude_sections: list | None = None, valid_section: callable = None
```

Comment:
Union `|` is not supported in python 3.9
```suggestion
    page: wikipedia.WikipediaPage, exclude_sections: list | None = None, valid_section: callable = None
```

Reply:
updated to py3.10